client/cmd/import.go

```diff
@@ -121,16 +122,18 @@ Currently, the only supported output format is %s.`,
 			return err
 		}
 
+		tokens := generator.Generate(scoreUpdates)
```

Comment:
Capturing our conversation on Slack:

We would like to break this apart a bit, so that the code generator produces an AST instead of a flat list of tokens. Along with that, we'd also like to make it so that the _parser_ produces an actual AST instead of a flat list of tokens. We are sort of already doing this, but it's entangled with the part where we turn the list of tokens into a list of ScoreUpdates. So one thing we'd like to do is refactor it so that there is an explicit step where we turn the list of tokens into an AST, and then a subsequent step where we turn the AST into a list of ScoreUpdates.

We would also implement the reverse, for the code generator: "unparsing" a list of ScoreUpdates into an AST.

Then the formatter would operate on the AST, turning it into a nicely formatted string of Alda code.